### DISCUSSION TITLE
[Raspberry Pi 3b] Sound output and sample editor

### DISCUSSION THREAD

**Discussion by mattgresh** [schismtracker/schismtracker#442] [Q&A] [answered]
Hello. Recently I built Schism from source code (20230713) on my Raspberry Pi (Raspbian "Buster" Lite). It works well but so far I've got some questions about its configuration and workflow.

1) On Raspberry typing by default command Schism forces all sound output via HDMI instead of headphones, no matter which one you enabled in `raspi-config`.  I googled that command `schismtracker -a alsa:default` eliminates the problem. But how to run Schism without typing this option every time?

2) Is there any possibility of sample editor to cut off excess part of loaded sample?

3) Where is the hotkey combination that will make me new Necros? :)

P.S.: I haven't found any forum which is dedicated for Schism Tracker. Considering that Github is not a platform where users ask their stupid questions I would like ask you to share a relevant one.

**Comment by mrpapersonic** [comment, marked as answer]
There is a configuration option (i.e. "Audio/driver") that does what you want. You can find more about that in the "Audio output" section of [docs/configuration.md](https://github.com/schismtracker/schismtracker/blob/master/docs/configuration.md).

I'm not sure of any way to cut samples in Schism (granted, most of my tracking is done in OpenMPT). If you want to edit samples you'll likely have to use a separate audio editor like Audacity.

You'll learn how to efficiently navigate around Schism (and Impulse) with time. It takes a lot of getting used to, especially with some of the fairly odd keybinds for today's standards (Alt+C and Alt+P are copy and paste respectively; Ctrl+C and Ctrl+V weren't as standardized when Impulse was created).

P.S. The Mod Archive has a forum that is general to all trackers, but I can't say for how active it is.

**Comment by mattgresh** [comment]
Thank you for the reply.

Messing with Schism perhaps for a month I even finished my first 1m soundtrack (ofc with no music school background so quality of the track is appropriate).
Solving audio driver glitch is still not obvious for me so I decided to leave it for a time and focused on tracking itself.
And yeah, can confirm my first post: as user I feel a bit a lack of sample editing functionality. In my opinion it would be much better to have dedicated window where it represents basic sample operations like cutting excessive part off, simple reverb processing (i.e. over sample, not separate effect command), low/high pass filter... I think it is really important for sample-oriented DAW when you can just modify the sound without testing it in some external sampler.
... I would even imagine how Polyend Tracker owners will feel themselves considering that their hardware trackers cost about $600 :)

Please don't take my comments as offensive or derogatory. Because only thanks to all involved Schism developers such dumb as me can touch the far echo of 1990s' Scene. 
Since I registered on modarchive forum I will ask my questions directly to community, not here. Thank you for the tip :)

[Edit]: Huh, I've found it! Cutting off can be achieved by positioning loop markers and hotkey Alt-L. All things are on F3 page ofc.
